test/testsimplifytokens.cpp

```diff
@@ -858,7 +858,7 @@ class TestSimplifyTokens : public TestFixture {
     unsigned int sizeofFromTokenizer(const char type[]) {
         Tokenizer tokenizer(&settings0, this);
         tokenizer.fillTypeSizes();
-        Token tok1(0);
+        Token tok1;
```

Comment:
I am not against that these redundant parameters are removed. But well.. I could live them also. Feel free to decide if you want to keep these changes or revert these.

Reply:
I think it would be more effort for me to revert the changes.  Let's remove the redundant parameters.